### DISCUSSION TITLE
Import in Account Failed

### DISCUSSION THREAD

**Discussion by up2sidd** [YetiForceCompany/YetiForceCRM#16777] [How to]
Hi! Any solution for this: -

After import it quickly redirects to 
![image](https://user-images.githubusercontent.com/108876771/210230958-0090315b-d846-4edb-9fd1-dd8adbabe703.png)

After checking on Accounts > Import, the below image appears and reloads only.

![image](https://user-images.githubusercontent.com/108876771/210229576-06428eeb-04ba-49aa-8015-cf2c35557e1d.png)

All the fields are on its respective place with proper heading. Have exported dummy entry and field with the previous database before importing.

Please help :(

**Comment by mariuszkrzaczkowski** [comment]
Check if CRON is working properly, and server configuration

**Reply by up2sidd** [reply]
Hi, thank you for the reply. Can you please suggest on the screen share below: -

![image](https://user-images.githubusercontent.com/108876771/210231666-3052bf86-9aeb-4ef8-8739-e054aedfc1f5.png)

**Reply by mariuszkrzaczkowski** [reply]
Attach the configuration report and why do you have a yellow triangle at the top?

**Reply by up2sidd** [reply]
The yellow triangle is kanban board option which I unknownly clicked and appears, as it need to pay, have hold it for now.

Here's the configuration you asked.
![ConfReport](https://user-images.githubusercontent.com/108876771/210234285-cb0b948e-198d-441c-887c-fa9421804a9a.png)

**Reply by mariuszkrzaczkowski** [reply]
fix what is red

**Reply by up2sidd** [reply]
Please show the way to fix those all . Can it be fixed on config.php or have to go multiple files?

**Reply by mariuszkrzaczkowski** [reply]
a lot to write this task for the server administrator
https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/web-server-requirements

**Comment by rskrzypczak** [comment]
I guess you have a problem with the format of the phone number in your import file (should consist of: "+", prefix, phone number. No whitespace or separators).
However, turn on [debugging](https://yetiforce.com/pl/component/zoo/item/debugging) and you will get more specific information about the cause of the error.

**Comment by s2b-git** [comment]
Check also emails columns in csv. Error in email address will stop import process
